core/src/main/scala/io/finch/Endpoint.scala

```diff
@@ -503,10 +504,10 @@ object Endpoint {
           case Throw(e) => Error.NotParsed(self.item, tag, e)
         }
 
-        if (errors.isEmpty)
-          Future.const(Try.collect(decoded).map(seq => NonEmptyList(seq.head, seq.tail.toList)))
-        else
-          Future.exception(Error.RequestErrors(errors))
+        NonEmptyList.fromList(errors) match {
+          case None => Future.const(Try.collect(decoded).map(seq => NonEmptyList(seq.head, seq.tail.toList)))
+          case Some(err) => Future.exception(Error.RequestErrors(Functor[NonEmptyList].widen(err)))
```

Comment:
This is because `Nel` is invariant in `A`. 

```
 found   : cats.data.NonEmptyList[io.finch.Error.NotParsed]
 required: cats.data.NonEmptyList[Throwable]
Note: io.finch.Error.NotParsed <: Throwable, but class NonEmptyList is invariant in type A.
You may wish to define A as +A instead. (SLS 4.5)
          case Some(err) => Future.exception(Error.RequestErrors(err))
```

Problably when we change the signature of `RequestErrors` from taking a `Throwable` to an `Error` this widen will be unneccesary, but I'm not entirely sure.